roles/keystone/tasks/k2k-sp.yml

```diff
@@ -4,24 +4,23 @@
 
 - name: install shibboleth private key
   template: src=etc/shibboleth/sp-key.pem dest=/etc/shibboleth/sp-key.pem
-            owner=_shibd group=_shibd mode=0600
+            owner=_shibd group=_shibd mode=0640
   notify: restart shibboleth
 
 - name: install shibboleth public certificate
   template: src=etc/shibboleth/sp-cert.pem dest=/etc/shibboleth/sp-cert.pem
-            owner=_shibd group=_shibd mode=0644
+            owner=_shibd group=_shibd mode=0640
   notify: restart shibboleth
 
 - name: configure shibboleth identity attributes
   template: src=etc/shibboleth/attribute-map.xml
-            dest=/etc/shibboleth/attribute-map.xml mode=0600
-            owner=_shibd group=_shibd
+            dest=/etc/shibboleth/attribute-map.xml mode=0644
   notify: restart shibboleth
 
 - name: configure shibboleth settings
-  template: src=etc/shibboleth/shibboleth2.xml
-            dest=/etc/shibboleth/shibboleth2.xml mode=0644
-            owner=_shibd group=_shibd
+  template:
+    src: etc/shibboleth/shibboleth2.xml
+    dest: /etc/shibboleth/shibboleth2.xml mode=0644
```

Comment:
This will break. You've got a key: value, and on the same line a key=value.  Your mode lines needs to move to it's own key: value line.